### ISSUE TITLE
Copy operation takes too much time while estimating `Time remaining`

### ISSUE THREAD

**Issue by shujaat-github** [miyabe/encfsy#15]
When I copy a **large file** to EncFSy drive, the copy operation takes too much time (around 5-15 seconds) while estimating **Time remaining** as can be seen in the screenshot given below. 

I noticed that the copy operation starts **instantly** when I copy the same file to an another partition or **Cryptomator** or **Boxcryptor** drives.

![image](https://github.com/mimidesunya/encfsy/assets/99479518/2b23ce0c-be4b-4d37-ba93-bda28ccdbe8e)

**Comment by shujaat-github** [comment]
@mimidesunya 

Does the last bug fix actually fixes this issue?

**Comment by mimidesunya** [comment]
Yep, it does. And I’m pretty sure it smoothed out a few other quirks too, meow! 🐾

**Comment by mimidesunya** [comment]
I’ve improved EncFSy’s stability, so this should be resolved as well, together with issues #11 and #14.
